static/app/views/insights/mobile/vitals/components/vitalScreens.tsx

```diff
@@ -0,0 +1,111 @@
+import styled from '@emotion/styled';
+
+import SearchBar from 'sentry/components/performance/searchBar';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {NewQuery} from 'sentry/types';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import useRouter from 'sentry/utils/useRouter';
+import useCrossPlatformProject from 'sentry/views/insights/mobile/common/queries/useCrossPlatformProject';
+import {getFreeTextFromQuery} from 'sentry/views/insights/mobile/screenload/components/screensView';
+import {useTableQuery} from 'sentry/views/insights/mobile/screenload/components/tables/screensTable';
+import {Referrer} from 'sentry/views/insights/mobile/ui/referrers';
+import VitalsScreensTable from 'sentry/views/insights/mobile/vitals/components/vitalsScreensTable';
+import {SpanMetricsField} from 'sentry/views/insights/types';
+import {prepareQueryForLandingPage} from 'sentry/views/performance/data';
+import {getTransactionSearchQuery} from 'sentry/views/performance/utils';
+
+export function VitalScreens() {
+  const router = useRouter();
+  const {selection} = usePageFilters();
+  const location = useLocation();
+  const organization = useOrganization();
+  const {isProjectCrossPlatform, selectedPlatform} = useCrossPlatformProject();
+
+  const {query: locationQuery} = location;
+
+  const query = new MutableSearch(['transaction.op:ui.load']);
+
+  const searchQuery = decodeScalar(locationQuery.query, '');
+  if (searchQuery) {
+    query.addStringFilter(prepareQueryForLandingPage(searchQuery, false));
+  }
+  if (isProjectCrossPlatform) {
+    query.addFilterValue('os.name', selectedPlatform);
+  }
+  const queryString = query.formatString();
+
+  const orderby = decodeScalar(locationQuery.sort, '-count');
+  const newQuery: NewQuery = {
+    name: '',
+    fields: [
+      SpanMetricsField.PROJECT_ID,
+      'transaction',
+      `count()`,
+      `avg(mobile.slow_frames)`,
+      `avg(mobile.frozen_frames)`,
+      `avg(mobile.frames_delay)`,
```

Comment:
Considering the sub-screens have release filters applied, should this page likewise query for the avg by release? Maybe not for this PR, but I think maybe drilling down into values and seeing the release filter appear (if it does, I wasn't able to test this on the `sentry-sdks` org yet) might be misleading to the user.

Reply:
Also in the future, are we more concerned about the slow frame _rate_ instead of the specific value? I know now the rate is more important because it normalizes the value across transaction durations

Reply:
Yes, fully agree - we should switch to frame rates. sub-task here: https://github.com/getsentry/sentry/issues/75318